akka-actor/src/main/scala/akka/actor/LightArrayRevolverScheduler.scala

```diff
@@ -191,7 +196,8 @@ class LightArrayRevolverScheduler(config: Config,
 
   @volatile private var timerThread: Thread = threadFactory.newThread(new Runnable {
 
-    var tick = 0
+    var tick = startTick
+    var tickWrap = 0L
```

Comment:
this deserves a comment here. What does this encode? What is the invariant between tick and tickWrap?